projects/Benchmarks/Networking/RabbitMqBroker.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading;
+using Ductus.FluentDocker.Builders;
+
+namespace Benchmarks.Networking
+{
+    public class RabbitMqBroker
+    {
+        public static IDisposable Start()
+        {
+            var broker = new Builder().UseContainer()
+               .UseImage("rabbitmq")
+               .ExposePort(5672, 5672)
+               .WaitForPort("5672/tcp", 40000 /*40s*/)
+               .ReuseIfExists()
+               .WithName("rabbitmq")
+               .Build()
+               .Start();
+
+            Thread.Sleep(5000); // the broker needs some more time after the port is open to be operational
```

Comment:
This approach is fine but you can also run `rabbitmqctl await_startup` and then wait for no more than a second or so.

Reply:
Failed to execute rabbitmqctl await_startup error: Error: unable to perform an operation on node 'rabbit@ed1d67aa8b5a'. Please see diagnostics information and suggestions below.
Most common reasons for this are:
 * Target node is unreachable (e.g. due to hostname resolution, TCP connection or firewall issues)
 * CLI tool fails to authenticate with the server (e.g. due to CLI tool's Erlang cookie not matching that of the server)
 * Target node is not running
In addition to the diagnostics info below:
 * See the CLI, clustering and networking guides on https://rabbitmq.com/documentation.html to learn more
 * Consult server logs on node rabbit@ed1d67aa8b5a
 * If target node is configured to use long node names, don't forget to use --longnames with CLI tools
DIAGNOSTICS
===========
attempted to contact: [rabbit@ed1d67aa8b5a]
rabbit@ed1d67aa8b5a:
  * connected to epmd (port 4369) on ed1d67aa8b5a
  * epmd reports: node 'rabbit' not running at all
                  other nodes on ed1d67aa8b5a: ['rabbitmqcli-47-rabbit']
  * suggestion: start the node
Current node details:
 * node name: 'rabbitmqcli-87-rabbit@ed1d67aa8b5a'
 * effective user's home directory: /var/lib/rabbitmq
 * Erlang cookie hash: Ga7ZD0sbg8lx30HyB3nk/A==

Reply:
the command does not seem to work

Reply:
setting a nodename fixed the issue